src/Compilers/VisualBasic/Portable/Binding/Binder_Invocation.vb

```diff
@@ -3009,6 +3043,9 @@ ProduceBoundNode:
                                 End If
 
                                 argumentNamesLocationsBuilder.Add(id.GetLocation())
+                            ElseIf argumentNamesBuilder IsNot Nothing Then
+                                argumentNamesBuilder.Add(Nothing)
+                                argumentNamesLocationsBuilder.Add(Nothing)
```

Comment:
I changed my mind about disallowing RangeArguments after named arguments in the parser. It looks like VB accepts RangeArguments in regular constructor calls without reporting any errors. Even though they are meant to be used in array allocations and ReDim statements. Let's make sure we do proper book-keeping and test scenarios involving them.